libretroshare/src/retroshare/rsgxschannels.h

```diff
@@ -706,21 +717,4 @@ class RsGxsChannels: public RsGxsIfaceHelper, public RsGxsCommentService
 	 */
 	RS_DEPRECATED_FOR(editChannel)
 	virtual bool updateGroup(uint32_t& token, RsGxsChannelGroup& group) = 0;
-
-	//////////////////////////////////////////////////////////////////////////////
-    ///                     Distant synchronisation methods                    ///
-    //////////////////////////////////////////////////////////////////////////////
-    ///
-    ///
-    // This approach is much cleaner than the "multicallback" system. We should keep it and use rsEvents to warn when
-    // new results are received.
-	virtual TurtleRequestId turtleGroupRequest(const RsGxsGroupId& group_id)=0;
```

Comment:
I dont understand why you would remove these methods. They are actually used, and even called from the GUI.